portal/portal-service-impl/impl/src/java/org/sakaiproject/portal/service/SiteNeighbourhoodServiceImpl.java

```diff
@@ -538,10 +534,7 @@ public String lookupSiteAlias(String id, String context)
 		{
 			if (aliases.size() > 1 && log.isInfoEnabled())
 			{
-				if (log.isDebugEnabled())
-				{
-					log.debug("More than one alias for "+ id+ " sorting.");
-				}
+                log.debug("More than one alias for {} sorting.", id);
```

Comment:
The indentation here looks wrong, could you try to match it with the surrounding context, the issue maybe because of differences tabs and spaces.